yarn.lock

```diff
@@ -2787,10 +2787,10 @@
     webcrypto-shim "^0.1.5"
     xhr2 "0.1.3"
 
-"@okta/okta-idx-js@0.22.0":
-  version "0.22.0"
-  resolved "https://registry.yarnpkg.com/@okta/okta-idx-js/-/okta-idx-js-0.22.0.tgz#c5dbebd155be2c8a8e2bbceb2e92701b6ba59349"
-  integrity sha512-SehJVwQI51xWrB1YaMn6UB54cOzhrM7z5k2xOI62I3B7qOQ3uFOZJvRCJnBcZ6TEcQuqJ3/6q93h6n8omw0tVQ==
+"@okta/okta-idx-js@0.24.0":
```

Comment:
is auth-js depending on idx-js ?

Reply:
yes. Eventually auth-js will consume idx-js and then idx-js will be retired. This widget refactor is a prerequisite for that effort.